airflow/logging_config.py

```diff
@@ -13,52 +13,38 @@
 # limitations under the License.
 #
 import logging
-import os
-import sys
-
 from logging.config import dictConfig
 from airflow import configuration as conf
 
-log = logging.getLogger(__name__)
-
+from airflow.utils.module_loading import import_string
 
-def configure_logging(custom_config_path=None):
-    if custom_config_path is not None:
-        config_path = custom_config_path
-    else:
-        config_path = conf.get('core', 'logging_config_path')
-
-    try:
-        path = os.path.dirname(config_path)
-        sys.path.append(path)
-
-        filename = os.path.basename(config_path).replace('.py', '')
+log = logging.getLogger(__name__)
 
-        logging_module = __import__(filename)
-        logging_config = logging_module.LOGGING_CONFIG
 
-        # Make sure that the variable is in scope
-        assert (isinstance(logging_config, dict))
+def configure_logging():
+    logging_class_path = conf.get("core", "logging_config_class")
 
-        log.info(
-            'Successfully imported user-defined logging config from %s',
-            config_path
-        )
-    except AttributeError:
-        # Import default logging configurations.
-        log.warn(
-            'Loaded custom config from %s, but could not find LOGGING_CONFIG, Using default config instead',
-            config_path
-        )
-        from airflow.config_templates.default_airflow_local_settings import \
-            DEFAULT_LOGGING_CONFIG as logging_config
-    except Exception:
-        # Import default logging configurations.
-        log.debug(
-            'Unable to load custom logging, using default config instead'
-        )
-        from airflow.config_templates.default_airflow_local_settings import \
+    if logging_class_path == '':
+        from airflow.config_templates.airflow_local_settings import \
```

Comment:
Please use () for multiline

Reply:
AFAIK this is not working for imports:
```
[GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.34)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> (from math import exp)
  File "<stdin>", line 1
    (from math import exp)
        ^
```

Reply:
The things to import go in the brackets:

```
from math import (
  exp as POW,
)
```

Reply:
Thanks, love it